pkgs/applications/audio/rhythmbox/default.nix

```diff
@@ -66,11 +67,27 @@ in stdenv.mkDerivation rec {
 
     gst_all_1.gstreamer
     gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
+    gst_all_1.gst-libav
 
     libdmapsharing_3 # necessary for daap support
+    libsecret
+    libnotify
   ] ++ gst_plugins;
 
-  configureFlags = [ "--enable-daap" ];
+  configureFlags = [
+    "--enable-daap"
+    "--enable-libnotify"
+    "--with-libsecret"
+  ];
+
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "$GST_PLUGIN_SYSTEM_PATH_1_0"
```

Comment:
This should be done automatically: https://github.com/NixOS/nixpkgs/blob/a68f9c2b77662562249f7a11a9cc736d0b28f6af/pkgs/build-support/setup-hooks/wrap-gapps-hook/wrap-gapps-hook.sh#L34

Reply:
That's right, and it is even mentioned in the manual. However many packages seem to do this explicitly (clementine, ...) so I was confused.

Reply:
It is only done for qt apps.